Documentation/compatibility/wpf - AppDomain shutdown handling may now invoke in cleanup of WeakEvents.md

```diff
@@ -0,0 +1,46 @@
+## WPF AppDomain Shutdown Handling May Now Invoke in Cleanup of Weak Events
+
+### Scope
+Edge
+
+### Version Introduced
+4.7.2
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+In .NET Framework 4.7.1 and earlier versions, WPF would potentially create a `System.Windows.Threading.Dispatcher` on the .NET finalizer thread during AppDomain shutdown.  This was fixed in 
+.NET Framework 4.7.2 and later versions by making the cleanup of weak events thread-aware.  Due to this, the cleanup may invoke from the finalizer thread to the correct 
```

Comment:
there's the same problem here: something has to be invoked

Reply:
I can clean this up as above.